### PR TITLE
Added details on the configuration options available for install of Umbraco default data.

### PR DESCRIPTION
**Pull request by AndyButland**
This updates the documentation section on configuration for some additional settings introduced by [this PR](https://github.com/umbraco/Umbraco-CMS/pull/12122), and expected in 9.5 (expected in six weeks time).

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @AndyButland ! 💪 

We'll make sure this is reviewed before the 9.5 release!

File: Reference/V9-Config/InstallDefaultDataSettings/index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+versionFrom: 9.0.0
+meta.Title: "Umbraco Install Default Data Settings"
+meta.Description: "Information on configuration allowing for the modification of default data installed in new projects"
+---
+
+# Install Default Data Settings
+
+When Umbraco is installed for the first time, it creates various default data.  These include a language, some data types and some media and member types.
```
Comment (eshanrnh) [inline Reference/V9-Config/InstallDefaultDataSettings/index.md:9]: data types, and some media

File: Reference/V9-Config/InstallDefaultDataSettings/index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+In certain setups you may want to take control over what is installed, and opt-out of the creation of certain items.
```
Comment (eshanrnh) [inline Reference/V9-Config/InstallDefaultDataSettings/index.md:11]: In certain setups, you may want to take control over what is installed and opt-out

File: Reference/V9-Config/InstallDefaultDataSettings/index.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+When working in a team and using a tool like Umbraco Deploy for sharing schema updates, when a colleague first sets up the project in their local environment, having this default data installed may not always be useful.  For example, if a different set of languages from Umbraco's default (en-US) have been set up, it would be preferable to not have the default recreated.  In other situations, certain Umbraco default data, member and media types may not be required.
+
+The following example configuration shows how this default data installation can be customised:
```
Comment (eshanrnh) [inline Reference/V9-Config/InstallDefaultDataSettings/index.md:15]: customized

**Review by eshanrnh — COMMENTED** [review]
Looks good, @AndyButland 💪 Just a couple of comments.

Also, kindly let me know the date when you want this to be merged/published 🙂

**Comment by AndyButland** [comment]
Thanks, have made the suggested amends.  This should go out with the release candidate of the CMS 9.5, which I _think_ is tomorrow... but I'll confirm.

**Comment by eshanrnh** [comment]
Merging it in 😀

**Pull request merged**